service/gcsutils/gcstools/netnsConfig.go

```diff
@@ -98,6 +98,13 @@ func netnsConfig() error {
 		return fmt.Errorf("netlink.LinkByName(%s) failed: %v", *ifStr, err)
 	}
 
+	// User requested non-default MTU size
+	if a.EncapOverhead != 0 {
+		if err = netlink.LinkSetMTU(link, int(a.EncapOverhead)); err != nil {
```

Comment:
Good catch. I misread the sample which had a very tricky `-=` not `=` when using `EncapOverhead`